kolibri_explore_plugin/assets/src/views/SearchPage.vue

```diff
@@ -3,67 +3,74 @@
     <DiscoveryNavBar />
-    <div class="flex-fill main">
+
+    <div
+      class="flex-fill"
+      :class="{ 'gray-background': !isEmpty, 'white-background': isEmpty }"
+    >
+
       <SearchBar
         v-model="query"
+        class="search-row"
         :debounce="800"
         :progress="progress"
         :loading="isLoading"
         @clear-input="clearInput"
       />
-      <b-container class="pb-5 pt-3">
-        <div v-if="isLoading" class="placeholder">
+
+      <div v-if="isEmpty" class="mt-5">
+        <template v-if="core.loading">
           <CardGridPlaceholder />
-        </div>
-        <div v-else>
-          <div v-if="isNoResults" class="empty mb-4 w-50">
-            <h1>Sorry, we can’t find any content that matches your search.</h1>
-            <h5>
-              You can try a different search, maybe use fewer words, or try one
-              of the topic suggestions below.
-            </h5>
-          </div>
-          <div v-if="isEmpty || isNoResults" class="align-items-center d-flex">
-            <h5 v-if="isEmpty" class="mb-0 mr-3 text-muted">
-              Topic Ideas
-            </h5>
-            <ButtonsBar
-              title="More Topics"
-              :buttons="Array.from(searchTerms.keys())"
-              @click="goToTerm"
-            />
-          </div>
-          <div v-else>
-            <Keywords :words="keywords" @click="removeKeyword" />
-          </div>
-        </div>
-      </b-container>
-
-      <div v-if="resultCards">
-        <div v-for="[kind, nodes] in resultCards" :key="kind">
-          <CardGrid
-            id="root"
-            variant="collapsible"
-            :itemsPerPage="4"
-            :nodes="nodes"
-            :mediaQuality="mediaQuality"
-            :cardColumns="cardColumns"
-          >
-            <div>
-              <h4 class="text-muted">
-                <span class="font-weight-normal">
-                  {{ nodes.length }} results for "{{ cleanedQuery }}"
-                </span>
-                to {{ groupVerb(kind) }}
-              </h4>
-            </div>
-          </CardGrid>
-          <hr>
-        </div>
+        </template>
+        <template v-else>
+          <ChannelsList />
+        </template>
       </div>
 
-      <b-container class="pb-5 pt-3">
-        <div v-if="resultChannels.length">
+      <template v-else>
+
+        <b-container class="pb-5 pt-3">
+          <template v-if="isLoading" class="placeholder">
+            <CardGridPlaceholder />
+          </template>
+          <template v-else>
+            <template v-if="!isEmpty">
+              <Keywords :words="keywords" @click="removeKeyword" />
+            </template>
+            <div v-if="isNoResults" class="empty mb-4 mt-5 w-50">
+              <h1>Sorry, we can’t find any content that matches your search.</h1>
+              <h5>
+                You can try a different search, maybe use fewer words, or try one
+                of the topic suggestions below.
```

Comment:
There's no topic suggestions below after this change, maybe we should change the wording. But it could be done as a separate task.